src/ets_lru/src/ets_lru.erl

```diff
@@ -366,4 +366,4 @@ table_name(Name, Ext) ->
 
 -spec strict_monotonic_time(atom()) -> strict_monotonic_time().
 strict_monotonic_time(TimeUnit) ->
-    {erlang:monotonic_time(TimeUnit), erlang:unique_integer([monotonic])}.
+    {erlang:monotonic_time(TimeUnit), make_ref()}.
```

Comment:
could we not pass in `Key` and return `{{erlang:monotonic_time(TimeUnit), Key}`? This makes it unique and is as cheap as possible.

Reply:
A ref is potentially a lot smaller than a key. A key currently could have a user name, a module and and salt and we'd copy that over to atimes and ctimes ets tables.

Reply:
hmm I guess